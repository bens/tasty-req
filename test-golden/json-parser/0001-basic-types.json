{ "a": null,
  "b": "some string",
  "c": 123,
  "d": 0.0,
  "e": 1E-3,
  "f": 0.321e10,
  "g": [1, 2.0, "3"],
  "h": { "x": 1, "y": 2 },
}
