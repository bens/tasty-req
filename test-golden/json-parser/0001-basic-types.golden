JsonObject
  (Object
     (fromList
        [ ( "a" , JsonNull )
        , ( "b" , JsonText "some string" )
        , ( "c" , JsonInteger 123 )
        , ( "d" , JsonDouble 0.0 )
        , ( "e" , JsonDouble 1.0e-3 )
        , ( "f" , JsonDouble 3.21e9 )
        , ( "g"
          , JsonArray [ JsonInteger 1 , JsonDouble 2.0 , JsonText "3" ]
          )
        , ( "h"
          , JsonObject
              (Object
                 (fromList [ ( "x" , JsonInteger 1 ) , ( "y" , JsonInteger 2 ) ]))
          )
        ]))
